lib/seeing-is-believing.coffee

```diff
@@ -32,98 +58,62 @@ module.exports = SeeingIsBelieving =
   deactivate: ->
     @subscriptions.dispose()
 
-  configDefaults:
-    'ruby-command':  'ruby'
-    'add-to-env':
-      'LANG':        defaultLang
-      'ADD_TO_PATH': ''
-    'flags': [
-      '--alignment-strategy', 'chunk',
-      '--number-of-captures', '300',
-      '--line-length',        '1000',
-      '--timeout',            '12'
-    ]
-
-  invokeSib: (vars) ->
-    editor        = vars.editor
+  invokeSib: (additionalArgs) ->
+    editor        = atom.workspace.getActivePaneItem()
```

Comment:
I'd rather pass the editor in than have the method go find the editor to operate on.